jvm-packages/xgboost4j-example/src/main/java/ml/dmlc/xgboost4j/java/example/BoostFromPrediction.java

```diff
@@ -32,18 +33,18 @@ public static void main(String[] args) throws XGBoostError {
     System.out.println("start running example to start from a initial prediction");
 
     // load file from text file, also binary buffer generated by xgboost4j
-    DMatrix trainMat = new DMatrix("../../demo/data/agaricus.txt.train");
-    DMatrix testMat = new DMatrix("../../demo/data/agaricus.txt.test");
+    DMatrix trainMat = new DMatrix("../demo/data/agaricus.txt.train");
+    DMatrix testMat = new DMatrix("../demo/data/agaricus.txt.test");
 
     //specify parameters
-    HashMap<String, Object> params = new HashMap<String, Object>();
+    Map<String, Object> params = new HashMap<>();
```

Comment:
No special reason, normally we should use interface rather than concrete class as the type?